src/System Application/App/ControlAddIns/src/BusinessChart.ControlAddin.al

```diff
@@ -0,0 +1,55 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Integration;
+
+controladdin BusinessChart
+{
+    RequestedHeight = 320;
+    MinimumHeight = 180;
+    RequestedWidth = 300;
+    MinimumWidth = 200;
+    VerticalStretch = true;
+    VerticalShrink = true;
+    HorizontalStretch = true;
+    HorizontalShrink = true;
+
+    Scripts = 'Resources\BusinessChart\js\BusinessChartAddIn.js',
+              'https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.5.1.min.js',
+              'https://code.highcharts.com/9.1.1/highcharts.js',
+              'https://code.highcharts.com/9.1.1/highcharts-more.js',
```

Comment:
How will those Control Add-Ins work in an OnPrem Scenario, that have limited access to any external ressources like CDNs?

And does this mean, that microsoft has aquired a license for highcharts and that we can use this library for custom control add-ins, too?

Reply:
This is a valid concern, I've created an internal work item to look into how to better accommodate OnPrem scenarios for these cases. Maybe these should be hosted locally or at least limited it to a single private CDN. For now, OnPrem environments with limited access to external resources will not benefit from these addins.

In regards to the license question I'm not too sure about that. This is a conversion of the old Business Chart control addin which was also using highcharts - so no new licenses have been acquired or any licensing changed as part of this PR. I'm not involved in the licensing part so please reach out through another channel if you want assistance with this inquiry.